cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -84,9 +84,10 @@ func TestAddFlagsFlag(t *testing.T) {
 			MinRequestTimeout:           1800,
 		},
 		Admission: &apiserveroptions.AdmissionOptions{
-			PluginNames: []string{"AlwaysDeny"},
-			ConfigFile:  "/admission-control-config",
-			Plugins:     s.Admission.Plugins,
+			RecommendedPluginOrder: []string{"NamespaceLifecycle"},
```

Comment:
Can you actually wire this by default in the sample-apiserver?

Reply:
Not sure if I understand. When this PR gets merged, by default (a cluster-admin did not provide plugin names) this plugin will be included in the `sample-apiserver`.

Reply:
> Not sure if I understand. When this PR gets merged, by default (a cluster-admin did not provide plugin names) this plugin will be included in the sample-apiserver.

Ok.  I read through below and now understand.  The genericapiserver sets a good default for users which do not care about admission in particular (or don't think they care).  Those servers will get better and better admission chains over time.  Servers that *do* care, can override after creation and they know that they care and should check during rebases.  This is consistent with our other options.  Good choice.

Can I get a comment explaining the default in the NewOptions method explaining the reasoning?